runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/util/PackageUtil.java

```diff
@@ -64,9 +64,13 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-/** Helper routines for packages. */
+/**
+ * <b><i>For internal use only; no backwards compatibility guarantees.</i></b>
+ *
+ * <p>Helper routines for packages.
+ */
 @Internal
-class PackageUtil implements Closeable {
+public class PackageUtil implements Closeable {
```

Comment:
We should not make this class public or its methods.

Reply:
Hmm, yes I first when directly to `PackageUtil` from the `DataflowRunner` but reverted that and used the stager directly. I'll revert this, too, though it doesn't matter since this whole directory shouldn't exist and is just an internal implementation detail.

Reply:
This still seems to be public

Reply:
Whoops. Thought I had simply dropped the commit but apparently botched that maneuver. Really done now.